### PR TITLE
Add support for \cancel and co.

### PR DESCRIPTION
**Pull request by Orace**
This commit address #349 and add support to:

- [x] `\cancel`
- [x] `\bcancel`
- [x] `\xcancel`
- [ ] `\cancelto`

Current output of `\cancel{Q} \\ \xcancel{69} \\ \frac{a\bcancel{b}}{\bcancel{b}}`:

cancel

### CODE REVIEW

File: src/XamlMath.Shared/Boxes/ZBox.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using XamlMath.Rendering;
+
+namespace XamlMath.Boxes
+{
+    public class ZBox : Box
```
Comment (ForNeVeR) [inline src/XamlMath.Shared/Boxes/ZBox.cs:6]: Maybe `LayeredBox`? What do you think? `ZBox` sounds kinda technical and wasn't obvious to me (I had to think for several seconds about that).

File: src/XamlMath.Shared/Boxes/ZBox.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        public override int GetLastFontId()
+        {
+            var fontId = TexFontUtilities.NoFontId;
+            foreach (var child in Children)
+            {
+                fontId = child.GetLastFontId();
+                if (fontId == TexFontUtilities.NoFontId)
+                    break;
+            }
+            return fontId;
```
Comment (ForNeVeR) [inline src/XamlMath.Shared/Boxes/ZBox.cs:35]: Is there a meaning to this logic? Otherwise, maybe just return the last child's font id?

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
Great work! There's a couple of minor issues left, could you please take a look?

I should note that this is a breaking change technically, but a good one and for a good reason, so I approve that.

**Comment by Orace** [comment]
@ForNeVeR, this is a draft work, I'm open to (and ask for) any advice on this.

File: src/XamlMath.Shared/Boxes/ZBox.cs (hunk 2 above)
Comment (Orace) [reply src/XamlMath.Shared/Boxes/ZBox.cs:35]: I don't even know what it's used for 🙄, I copied it from `HorizontalBox`.

File: src/XamlMath.Shared/Boxes/ZBox.cs (hunk 1 above)
Comment (Orace) [reply src/XamlMath.Shared/Boxes/ZBox.cs]: Fixed in e9e1e1e.

File: src/XamlMath.Shared/Boxes/ZBox.cs (hunk 2 above)
Comment (ForNeVeR) [reply src/XamlMath.Shared/Boxes/ZBox.cs]: Aha, okay. Let's keep it as it is for now.

**Comment by ForNeVeR** [comment]
I think we'll leave `\cancelto` still open for grabs for now.

**Pull request merged**